src/main/java/org/usfirst/frc/team1758/robot/commands/DriveWithJoystick.java

```diff
@@ -6,16 +6,20 @@
 public class DriveWithJoystick extends CommandBase {
 	private static boolean finished;
 	public DriveWithJoystick() {
-		//requires(driveTrain);
+		requires(driveTrain);
+		requires(sensors);
 	}
 	protected void initialize() {
 		finished = false;
 	}
 	protected void execute() {
 		//If you need to do calculations on the Axis do them in a new method inside the Controller
-		double x = OI.drivingController.getRawAxis(Axes.LEFT_X);
-		double y = OI.drivingController.getRawAxis(Axes.LEFT_Y);
-		driveTrain.setPower(x, y);
+		double x = OI.drivingController.getNormalizedAxis(Axes.LEFT_X);
+		double y = OI.drivingController.getNormalizedAxis(Axes.LEFT_Y);
+		double twist = OI.drivingController.getTwist();
+		double gyroAngle = sensors.getGyroAngle();
+		driveTrain.mecanumDrive(x, y, twist, gyroAngle);
+		//driveTrain.tankDrive(x, y);
```

Comment:
Remove commented out line. We don't need this. Git keeps the history of the files we push up.